.github/ISSUE_TEMPLATE/feature.md

```diff
@@ -10,10 +10,13 @@ assignees: ''
 Please take the time to describe the issue in detail, so that other contributors can pick it up and implement or declare it done/obsolete in case if was completed out of band.
```

Comment:
Maybe just "Describe..."

Reply:
Would rather keep this one, same as for the bug report - too uninviting otherwise.